input-events/idlharness.html

```diff
@@ -0,0 +1,40 @@
+<title>Input Event IDL tests</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/resources/WebIDLParser.js"></script>
+<script src="/resources/idlharness.js"></script>
+
+<pre id="untested_idl">
+interface InputEventInit {
+};
+
+interface UIEvent {
+};
+
+interface StaticRange {
+};
+</pre>
+
+<pre id='idl'>
+[Constructor(DOMString type, optional InputEventInit eventInitDict)]
+interface InputEvent : UIEvent {
```

Comment:
Yeah, testing the main interface under uievents/interface sounds right.

Note that idlharness.js doesn't actually try invoking any constructors, so one still has to do things like `assert_true(new Event('type', { bubbles: true}).bubbles)` as separate tests, to ensure that the dictionaries are actually hooked up correctly.